src/Esfa.Roaao.Xslx.IntegrationTests/RoaaoExcelTests.cs

```diff
@@ -154,15 +154,18 @@ private bool CheckWebsiteLink(string website)
                     request.Headers.Add("Cache-Control", "no-cache");
                     request.Headers.Add("User-Agent", "Mozilla/5.0 (Windows NT 6.2; WOW64; rv:19.0) Gecko/20100101 Firefox/19.0");
                     ServicePointManager.SecurityProtocol = SecurityProtocolType.Ssl3 | SecurityProtocolType.Tls | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12;
-                    using (var response = httpClient.SendAsync(request))
+                    for (var i=0; i<=2; i++)
```

Comment:
why a for loop? for retry logic we use Polly

Reply:
because it's a test, and a packaged one, it's probably better to do the retry with just loops

Reply:
does it make any sense to send two requests without waiting some time in between? maybe we should add some wait time if the status code is not 200

Reply:
Yes, I discussed this with Arvinth, and he was concerned introducing waiting in tests would build the test time to something significant (not just one here and there obviously, but if it becomes a pattern)